src/app/courses/cousers-icon.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'planet-course-icon',
+  template: `
+  <ng-container [ngSwitch]="icon">
+    <span *ngIf="step?.exam?.questions.length" i18n-matTooltip matTooltip="The test has question">
+      <mat-icon *ngSwitchCase="'assignment'">assignment</mat-icon>
+    </span>
+    <span *ngIf="step?.resources?.length" i18n-matTooltip matTooltip= "There are resource.">
```

Comment:
Please in the HTML templates make sure there is no space between the `=` and `"`